client.go

```diff
@@ -1989,19 +1990,35 @@ func (c *Client) GetMigrationSourceWS(container string) (*Response, error) {
 	return c.post(url, body, Async)
 }
 
-func (c *Client) MigrateFrom(name string, operation string, certificate string, secrets map[string]string, architecture string, config map[string]string, devices shared.Devices, profiles []string, baseImage string, ephemeral bool) (*Response, error) {
+func (c *Client) MigrateFrom(name string, operation string, certificate string,
```

Comment:
Yeah, no big deal either way, since we are breaking this function signature anyway. We just shouldn't start doing the same kind of change to the others (exec comes to mind) so we don't end up breaking downstreams multiple times in a row when not strictly needed.